code/_onclick/hud/health_adv.dm

```diff
@@ -0,0 +1,37 @@
+/obj/screen/adv_health
+	name = "YOU SHOULD NOT SEE THIS!"
+	icon = 'goon/icons/mob/mhealth.dmi'
```

Comment:
get a non goon sprite or get permission from the artist so we don't add more stuff to the goon folder

Reply:
who tf is the artist

Reply:
just make ur own sprites stupid cat

Reply:
Did you guys realize Gannets posted in here

We can just ask him
@Gannets can we use your death laser while we work on our own

Reply:
Wait shit wrong pr this isnt the laser beam

Reply:
Oh dear. Yeah, you're welcome to, pretty sure someone wandered in from Hippie to ask if they could use it a while back.

Also, not sure who drew the health monitor icons, I'd take a stab and assume Haine?

Reply:
There is literally nothing wrong with goonsprites, ignore the license autism Morty @MoreRobustThanYou